src/BlobBasics.java

```diff
@@ -228,37 +212,38 @@ private static void basicBlockBlobOperations(CloudBlobContainer container) throw
 
         // Upload a sample file as a block blob
         System.out.println("\n\tUpload a sample file as a block blob.");
-        CloudBlockBlob blockBlob1 = container.getBlockBlobReference("blockblob1.tmp");
+        BlobClient blockBlob1 = container.getBlobClient("blockblob1.tmp");
```

Comment:
"client" variable names.

Reply:
The variable name has been  changed to [ blobClient] according to your suggestion